blivet/formats/fs.py

```diff
@@ -941,7 +947,7 @@ class EFIFS(FATFS):
     _min_size = Size("50 MiB")
     _check = True
     _mount_class = fsmount.EFIFSMount
-    parted_flag = PARTITION_BOOT
+    parted_flags = PARTITION_ESP
```

Comment:
Is there a reason to change name of the variable from `parted_flag` to `parted_flags`?

Reply:
Fixed. That's from the first iteration where I tried to set both `PARTITION_BOOT` and `PARTITION_ESP`.